docs/apm/real-user-monitoring/dashboards.md

```diff
@@ -29,32 +29,38 @@ The data will populate in your organization's **Sumo Logic RUM - default** das
 
 Once the RUM app has been installed, use our [Explore view](/docs/dashboards-new/explore-view.md) to gain visibility into your web app's performance and end-user activity, such as geographic location, browser type, operating systems used. These dashboards visualize RUM metrics gathered from browser tracing instrumentation.
 
-Explore organizes RUM data on four levels:
-* **Application**: corresponds to the value of the application tag set in your RUM JavaScript script above. This should correspond to your whole website defined by its business function, such as "Coffee shop".
-* **Service**: corresponds to the name of the service in your RUM JavaScript script above. This should correspond to a JS code executed in the browser, such as "coffee-shop-web". You can have multiple services for each application. 
-* **Action Type**: can be one of:
-   * **document loads**: representing loading of actual documents and their resources into the browser
-   * **XHR actions**: representing any interaction with a page like click or submit that executes AJAX requests in the background to communicate with the backend, or
-   * **route changes**: single-page-app specific way to navigate to a new page/view without having to load a new document.
-* **Action Name**: automatically generated from URLs. No configuration is required. The specifics of it will depend on action type.
-
-Action names can contain asterisks (`*`) to replace automatically-detected dynamic parts of the URL. If you have action names that overlap, the action name with an asterisk contains data for page loads NOT contained in more specific action names:
-
-For example, `http://www.site.com/path/page.htm` does not contain actions from `http://www.site.com/path/*`.
-
-There are three dashboard types on the **Application** and **Service** level and a single one on the **Action type** and **Action** level. You can select the appropriate dashboard from dropdown menu in the header:<br/>![explore rum with red box.png](/img/rum/explore-rum-with-red-box.png)
+1. Go to **+New** > **Explore**.
+1. Under **Explore By**, **click Real User Monitoring**.
+1. Select your desired dashboard from **Dashboard** dropdown menu in the header:<br/>![explore rum with red box.png](/img/rum/explore-rum-with-red-box.png)<br/> There are three dashboard types on the **Application** and **Service** level and a single one on the **Action type** and **Action** level.
```

Comment:
There are three dashboard types on the **Application**, **Service** and **Environment** levels

Reply:
thank you!